ocf_data_sampler/config/model.py

```diff
@@ -102,6 +102,44 @@ class TimeResolutionMixin(Base):
     )
 
 
+class Sites(DataSourceMixin, TimeResolutionMixin, DropoutMixin):
+    """Site configuration model"""
+
+    filename: str = Field(
+        ...,
+        description="The NetCDF files holding the power timeseries.",
+    )
+    metadata_filename: str = Field(
+        ...,
+        description="The CSV files describing wind system.",
+    )
+
+    # site_ids: List[int] = Field(
+    #     None,
+    #     description="List of the ML IDs of the Wind systems you'd like to filter to.",
+    # )
+
+    @field_validator("forecast_minutes")
+    def forecast_minutes_divide_by_time_resolution(cls, v: int, info: ValidationInfo) -> int:
+        """Check forecast length requested will give stable number of timesteps"""
+        if v % info.data["time_resolution_minutes"] != 0:
+            message = "Forecast duration must be divisible by time resolution"
+            logger.error(message)
+            raise Exception(message)
+        return v
+
+    @field_validator("history_minutes")
+    def history_minutes_divide_by_time_resolution(cls, v: int, info: ValidationInfo) -> int:
+        """Check history length requested will give stable number of timesteps"""
+        if v % info.data["time_resolution_minutes"] != 0:
+            message = "History duration must be divisible by time resolution"
+            logger.error(message)
+            raise Exception(message)
+        return v
+
+    # TODO validate the netcdf for sites
+    # TODO validate the csv for metadata
```

Comment:
either do, or make into issue

Reply:
I think this needs the power system structure finalised first? So can be done later

Reply:
yea this can be done in #46 . Ill like to keep the TODOs in as a reminder

Reply:
So in the #46 discussion I think we already agreed what the schema should look like for both the power file and site_metadata file, this file could perhaps be the place to document what that schema should look like, or it could be in the `site.py` load file? 

Happy to add some validation in another PR, checking the required fields are there and the power is in correct units for example.